pyqir/setup.cfg

```diff
@@ -0,0 +1,39 @@
+[metadata]
+name = pyqir
+version = 0.3.0a1
+author = Microsoft
+description = PyQIR - set of APIs for generating, parsing, and evaluating Quantum Intermediate Representation (QIR)
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/qir-alliance/pyqir
+project_urls =
+	Bug Tracker = https://github.com/qir-alliance/pyqir
+classifiers =
+	Development Status :: 3 - Alpha
+	Intended Audience :: Developers
+	License :: OSI Approved :: MIT License
+	Programming Language :: Python :: 3
+	Programming Language :: Python :: 3.6
+	Programming Language :: Python :: 3.7
+	Programming Language :: Python :: 3.8
+	Programming Language :: Python :: 3.9
+	Programming Language :: Python :: 3.10
+	Programming Language :: Python
+	Operating System :: Microsoft
+	Operating System :: MacOS
+	Operating System :: Unix
+
+[options]
+package_dir =
+	= .
```

Comment:
Is this a typo?

Reply:
@samarsha Thanks! Fixed along with license def in 7e3e280